tensorflow_model_optimization/python/core/clustering/keras/cluster_config.py

```diff
@@ -18,6 +18,17 @@
 
 
 class CentroidInitialization(str, enum.Enum):
+  """Specifies how the cluster centroids should be initialized.
```

Comment:
Question: did you run the [linter](https://github.com/tensorflow/model-optimization/blob/master/CONTRIBUTING.md)? The internal version of that linter gave a suggestion to
add a blank line after the one-line summary docstring. If you did and this warning didn't show up, then will have to look into why the the external linter is missing this.